api/database/migrations/2022_06_30_182650_update_pool_with_advertisement.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class UpdatePoolWithAdvertisement extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('pools', function (Blueprint $table) {
+            $table->timestamptz('expiry_date')->nullable();
+        });
+
+        Schema::table('pools', function (Blueprint $table) {
```

Comment:
All the schema changes for pools here can be done in the same function similar to the ones for `pools_essential_skills` and `pools_essential_skills`

Reply:
yes!

Reply:
Beautiful!